parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -137,13 +137,15 @@ installMoby() {
 installKataContainersRuntime() {
     # TODO incorporate this into packer CI so that it is pre-baked into the VHD image
     echo "Adding Kata Containers repository key..."
+    ARCH=x86_64
```

Comment:
can we do `ARCH=$(arch)` to make this more reusable?

Reply:
Good point, I'll update this now.